docs/src/pages/components/badges/AccessibleBadges.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import Badge from '@material-ui/core/Badge';
+import MailIcon from '@material-ui/icons/Mail';
+
+export default function SimpleBadge() {
+  const notificationsCount = 998;
+  function notificationsLabel(count) {
+    if (count === 0) {
+      return 'notifications';
+    }
+    if (count > 99) {
+      return '99 plus notifications';
+    }
+    return `${count} notifications`;
+  }
+
+  return (
+    <Box
+      sx={{
+        '& > *': {
+          margin: 2,
+        },
+      }}
+    >
+      <a aria-label={notificationsLabel(notificationsCount)} href="/notifications">
+        <Badge
+          aria-labelledby="badge"
+          aria-describedby="notificationCount"
```

Comment:
What is the desired behavior here? The link will not use this naming (`aria-label`). The icon will not use it either (`titleAccess`)

Reply:
Just wanted to show use of both attributes. Will remove aria-label from link and titleAccess from icon.

Reply:
> Just wanted to show use of both attributes.

But what would this be used for? How would assistive technology behave if differently if these attributes were added?

Reply:
This will add label and description to badge component. I don't have much knowledge about aria will read about it carefully and get back to this soon.